guidelines/index.html

```diff
@@ -4198,7 +4198,74 @@ <h2 id="x7-input-purposes-for-user-interface-components"><span class="secno">7.
+            </li>
+            					
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus">1.4.13 Content on Hover or Focus</a>, the word "dismissable" has been changed to "dismissible".
+            </li>
+            					
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#robust">4. Robust</a>, repetition of the word "by" has been removed.
+            </li>
+            					
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#cc2">5.2.2 Full pages</a>, the third note began with "New" which has been removed.
+            </li>
+            					
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#conformance-required">5.3.1 Required Components of a Conformance Claim</a> the editorial note "In WCAG 2.0 this was a dated URI, which may need to be adjusted
+               when this becomes a Rec." has been removed.
+            </li>
+            					
+            <li>In the definition for <a href="https://www.w3.org/TR/WCAG21/#dfn-keyboard-interface">keyboard interface</a>, the second (of three) note has been changed to be an example of the first note,
+               leaving only two actual notes.
+            </li>
+            					
+            <li>In the definition for <a href="https://www.w3.org/TR/WCAG21/#dfn-technologies">technology</a>, the third note has been changed to be an example.
+            </li>
+            					
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#input-purposes">7. Input Purposes for User Interface Components</a>, the word "county" has been changed to "country".
+            </li>
+            					
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#orientation">1.3.4 Orientation</a>, the note referencing "binary display orientation" has been clarified to read "content
+               is not necessarily restricted to landscape or portrait display orientation".
+            </li>
+            			
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#issue-container-generatedID-43">a note in the definition of accessibility supported</a>, references to "Conformance Criterion" were changed to "Conformance Requirement".
+            </li>
+            					
+            <li>In the <a href="https://www.w3.org/TR/WCAG21/#dfn-relative-luminance">definition of relative luminance</a>, the red threshold was updated from 0.03928 to 0.04045.
+            </li>
+            
+            <li>In <a href="https://www.w3.org/TR/WCAG21/#parsing">4.1.1 Parsing</a> one note was deleted, and two notes were added, including: "This Success Criterion
+               should be considered as always satisfied for any content using HTML or XML."
+            </li> 
+            				
+         </ul>
+         		
+	<p>WCAG 2.1 に対する完全な<a href="https://github.com/w3c/wcag/commits/WCAG-2.1/guidelines/index.html">コミット履歴</a>も参照可能である。</p>
```

Comment:
原文では "commit history to WCAG 2.1" がリンクテキストになっていますので、リンクテキストに「WCAG 2.1」も含めた方がいいかもしれません。こうでしょうか?

```suggestion
	<p>完全な<a href="https://github.com/w3c/wcag/commits/WCAG-2.1/guidelines/index.html">WCAG 2.1 に対するコミット履歴</a>も参照可能である。</p>
```

Reply:
おっしゃるとおりです！ご指摘ありがとうございます。